src/Footer/Footer.jsx

```diff
@@ -0,0 +1,41 @@
+import "./footer.css";
+
+export const Footer = () => {
+  return (
+    <>
+      <a href="#" className="btn btn-float-action" id="back-to-top">
+        <i className="fas fa-arrow-up"></i>
+      </a>

```

Comment:
Now you change a tags to Link tag and routing too

Reply:
Yes, I will add router in next version.